client/src/resources/lang/en_us.json

```diff
@@ -622,8 +622,13 @@
     "chatMessage.yourConvertFailed": "You failed to convert your target.",
 
     "chatMessage.youDied":"You died!",
-    "chatMessage.youSurvivedAttack": "You were attacked but had defense and survived.",
-    "chatMessage.someoneSurvivedYourAttack": "Someone you attacked had defense and survived.",
+    "chatMessage.youSurvivedAttack": "You were attacked by someone, but you had defense and survived.",
+    "chatMessage.someoneSurvivedYourAttack": "You attacked someone, but they had defense and survived.",
+    "chatMessage.youAttackedSomeone": "You attacked someone.",

```

Comment:
I think it would still be nice to have some variants like `chatMessage.youAttackedSomeone.engineer` just because this would probably confuse me as a player